File: Data Structures & Algorithms/find-all-numbers-disappeared-in-an-array/submission-1.go
func findDisappearedNumbers(nums []int) []int {
	freqMap := make([]int,100001)
	result := make([]int,0)

	for _,val := range nums {
		freqMap[val]++
	}

	for i := 1; i <= len(nums); i++{
		if freqMap[i] == 0 {
			result = append(result, i)
		}
	}

	return result
}

File: Data Structures & Algorithms/sign-of-the-product-of-an-array/submission-1.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        count = nums[0]
        for i in range(1,len(nums)):
            count *= nums[i]
            if count == 0:
                return 0

        if count > 0:
            return 1

        return -1